libs/common/src/components/screens/welcome/index.tsx

```diff
@@ -101,10 +103,22 @@ export const Welcome = observer<WelcomeProps>(function Welcome({
   const theme = useTheme();
 
   const accountPickerModalProps = useAccountPickerModalProps();
+  const allowedButtons = theme.welcome?.buttons ?? [];
+  const isAllowed = (button: WelcomeButtons) => allowedButtons.includes(button);
 
   return (
     <WelcomeLayout title={renderTitle()} subTitle={renderSubTitle()}>
-      {walletsStore.wallets.length > 0 ? (
+      {isAllowed(WelcomeButtons.Zepeto) && <ZepetoButton />}
+      {isAllowed(WelcomeButtons.NewUser) && walletsStore.wallets.length > 0 && (
```

Comment:
That if doesn't make sense. Why only show `New User` if there are > 0 wallets?

Reply:
Also: this should probably be merged with `GetStarted` since I assume it will always have the same behavior.